### ISSUE TITLE
Proposal abstract

### ISSUE THREAD

**Issue by nmarble** [chrisvarga/chatbot#2]
Write abstract for Proposal Paper

References here: #8

**Comment by nmarble** [comment]
Abstract - Chatbots are computer programs designed to attempt to emulate a type of intelligent conversation that can be indistinguishable to, or equivalent to that which a human could provide. They are used for a multitude of functions from technical assistance to personal companionships. A large variety of these chatbots use large databases with collected hard coded answers or attempt to learn proper answers through some sort of machine learning. Thus, this approach can sometimes create reasonable but unhuman like responses. We propose to instead use actual human responses derived from Twitter to simulate a more human grammar.

**Comment by chrisvarga** [comment]
> Chatbots are computer programs designed to attempt to emulate a type of intelligent conversation that can be indistinguishable to, or equivalent to that which a human could provide.

This sentence is a little wordy. Maybe reword to something like:

Chatbots are computer programs designed to emulate an intelligent conversation that is indistinguishable to what a human could provide.

**Comment by nmarble** [comment]
I concur, Updated
